ospfd/ospf_packet.c

```diff
@@ -2427,6 +2430,11 @@ static int ospf_check_network_mask(struct ospf_interface *oi,
 	    || oi->type == OSPF_IFTYPE_VIRTUALLINK)
 		return 1;
 
+	/* Ignore mask check for max prefix length (32) */
+	if (oi->type == OSPF_IFTYPE_POINTOMULTIPOINT
+	    && oi->address->prefixlen == IPV4_MAX_BITLEN)
+		return 1;
+
```

Comment:
@pguibert6WIND this change is added to ignore network mask check for interface is configured with /32 prefix

Reply:
got it. my comment was about the comit log title prefix;